go.mod

```diff
@@ -71,7 +71,10 @@ require (
 	github.com/robfig/cron/v3 v3.0.1
```

Comment:
not quite the correct line, but after merging please check in checkmarx if any extra ignores need to be added